[ENH] add reports section / GSOD 2024 BIDS-INCF report (#593)
bids-standard/bids-website: docs/collaboration/reports/google_summer/2024-gsod.md

   </td>
   <td>Re-scoping project can be okay to solve challenges in the most effective way
   </td>
  </tr>
</table>

#### What could be improved?

<table>
  <tr>
   <td><strong>Topic</strong>
   </td>
   <td><strong>What we did</strong>
   </td>
   <td><strong>Lesson learned</strong>
   </td>
  </tr>
  <tr>
   <td>Budget
   </td>
   <td>1 writer stipend; 3 mentor stipends split across 4.
   </td>
   <td>We underestimated mentor effort and would budget more mentor stipends
   </td>
  </tr>
  <tr>
   <td>Communication
   </td>
   <td>Mattermost plus alternating weekly check-in/coworking times
   </td>
   <td>Allocate extra mentor time to managing and setting timelines
   </td>
  </tr>
  <tr>
   <td>Onboarding
   </td>
   <td>Set a loose order of on-boarding tasks
   </td>
   <td>Encourage an order of completion for early tasks e.g. friction log
   </td>
  </tr>
  <tr>
   <td>Project Management
   </td>
   <td>GitHub Project Board
   </td>
   <td>It took a few iterations to get this right. Deliverable timelines between meetings should be set.
   </td>
  </tr>
</table>

> 
> delete
> delete
> delete

Reviewer: ```suggestion
```

Reviewer: last push

# Analysis

This was a successful project. The project resulted in positive user feedback about the changes implemented by only 60% through the timeline.

During on-boarding as we began breaking down deliverables into tasks, the mentors realized that a few deliverables could be resolved most effectively by mentors managing the most technical steps. This minor re-scoping helped move the project forward quickly. Some timelines were delayed due to outside factors during the course of the project and timelines were adjusted accordingly.

The project experience underscored the value of external support for open-source initiatives – both in terms of funding and umbrella organization administration.

## Lessons Learned

> 
> 
> 

Reviewer: ```suggestion

```
delete extra spacing

   </td>
   <td>Re-scoping project can be okay to solve challenges in the most effective way
   </td>
  </tr>
</table>

#### What could be improved?

<table>
  <tr>
   <td><strong>Topic</strong>
   </td>
   <td><strong>What we did</strong>
   </td>
   <td><strong>Lesson learned</strong>
   </td>
  </tr>
  <tr>
   <td>Budget
   </td>
   <td>1 writer stipend; 3 mentor stipends split across 4.
   </td>
   <td>We underestimated mentor effort and would budget more mentor stipends
   </td>
  </tr>
  <tr>
   <td>Communication
   </td>
   <td>Mattermost plus alternating weekly check-in/coworking times
   </td>
   <td>Allocate extra mentor time to managing and setting timelines
   </td>
  </tr>
  <tr>
   <td>Onboarding
   </td>
   <td>Set a loose order of on-boarding tasks
   </td>
   <td>Encourage an order of completion for early tasks e.g. friction log
   </td>
  </tr>
  <tr>
   <td>Project Management
   </td>
   <td>GitHub Project Board
   </td>
   <td>It took a few iterations to get this right. Deliverable timelines between meetings should be set.
   </td>
  </tr>
</table>

> 
> ```delete
> ```delete
> ```delete

Reviewer: ```suggestion

```delete
```delete
```delete
```
delete again

> 
> ```delete
> ```delete
> ```delete

Reviewer: ```suggestion

```delete
```delete
```delete
```
delete en mass

# BIDS - INCF 2024 Google Season of Docs Case Study

## Umbrella Organization [INCF](incf.org) International Neuroinformatics  Coordinating Facility

> ## Project Organization	[BIDS](http://incf.org) Brain Imaging Data Structure

Reviewer: ```suggestion
## Project Organization	[BIDS](https://bids.neuroimaging.io/) Brain Imaging Data Structure
```

## Project Title 	[Streamlining the BIDS Online Presence](https://www.incf.org/activities/gsod)

> About BIDS  \

Reviewer: ```suggestion
#### About BIDS  
```

> The Brain Imaging Data Structure (BIDS) is an open global community driving the standardization of neuroscience data across a broad and growing range of modalities and health research disciplines. First released in June 2016, it is supported by a worldwide research network and endorsed by organizations like the International Neuroinformatics Coordinating Facility (INCF). A very large community, including ~29K yearly visitors, use the BIDS specification. More than 300 BIDS contributors currently support and maintain the BIDS community resources to structure and share their data. BIDS encompasses over 40 domain-specific and modality-specific technical specifications, open software conversion and analytics tools, and global infrastructure for collaborating on emerging standards in neuroscience ([Poldrack, et al, 2024](https://doi.org/10.1162/imag_a_00103)).

Reviewer: ```suggestion
The Brain Imaging Data Structure (BIDS) is an open global community driving the standardization of neuroscience data across a broad and growing range of modalities and health research disciplines. First released in June 2016, it is supported by a worldwide research network and endorsed by organizations like the International Neuroinformatics Coordinating Facility (INCF). A very large community, including ~29K yearly visitors, use the BIDS specification. More than 300 BIDS contributors currently support and maintain the BIDS community resources to structure and share their data. BIDS encompasses over 40 domain-specific and modality-specific technical specifications, open software conversion and analytics tools, and global infrastructure for collaborating on emerging standards in neuroscience ([Poldrack 2024](https://doi.org/10.1162/imag_a_00103)).
```

Case Study Authors		Christine Rogers &lt;[christine.rogers@mcgill.ca](mailto:christine.rogers@mcgill.ca)>

>                 Eric Earl &lt;[eric.earl@nih.gov](mailto:eric.earl@nih.gov)>

Reviewer: ```suggestion
Eric Earl
```

> 
> 
>                 on behalf of the project participants

Reviewer: ```suggestion
on behalf of the project participants
```

# Summary

> 
> *Fill out the following table and provide brief answers to the questions. You can provide more details in later sections.** We recommend you write out the detailed sections first then come back to write the summary.***

Reviewer: ```suggestion
```
deleting instructions

<table>
  <tr>
   <td><strong># Tech Writers</strong>
   </td>
   <td><strong>TW Project Hours</strong>
   </td>
   <td><strong>Budget</strong>
   </td>
   <td><strong>% Project Completed</strong>
   </td>
  </tr>
  <tr>
   <td>1
   </td>
   <td>150
   </td>
   <td>$6,500 USD
   </td>
   <td>100%
   </td>
  </tr>
</table>

> 
> 
> 
> * What problem were you trying to solve? And how did you try to solve it?
> * What were the outcomes of your project?
> * What went well? And what would you do differently?
> * What advice would you give to other projects trying to solve a similar problem with documentation?
> 

Reviewer: ```suggestion
```
deleting instructions

   <td>
   </td>
   <td>Giscus at the bottom of the front page was re-scoped and implemented by a mentor
   </td>
  </tr>
</table>

The above table summarizes the many changes across several repositories (PRs linked here):

* [BIDS Website repo](https://github.com/bids-standard/bids-website/pulls?q=+is%3Apr+author%3Aiamdamion) and sub/related repos with content:
* [BIDS-Apps Repo](https://github.com/bids-apps/bids-apps.github.io/pull/140)
* [BIDS Execution Spec](https://github.com/bids-standard/execution-spec/pull/24)
* [BIDS Extensions repo](https://github.com/bids-standard/bids-extensions/pull/35)

#### Unplanned Deliverables

*Were there any deliverables created that were not planned at the start of the project? If so, please explain.*

No unplanned deliverables arose in the course of this project.

## Metrics

*What metrics did you choose to measure the success of the project? Were you able to collect those metrics? Did you observe any change as a result of the project? Did you add or remove any metrics since your proposal? How often do you intend to collect metrics going forward?*

***Survey data — What can we glean from the October survey vs. the April survey? ***

> We performed a broad-reaching public BIDS community survey (120 respondents) before this project started and a narrower website-specific survey at the 60% mark (19 respondents) in this project’s timeline. The results were overwhelmingly positive. For instance, when asked to rate the overall utility of the old vs the new website, it is was overwhelmingly rated as “OK” (nearly two-thirds of respondents on the scale: Bad, OK, Good, or Great). Some of the improvements mentioned were: acceleration in accessing info, Y was appreciated and Z demonstrated that this project helped significantly in clarifying understanding of the material. * *

Reviewer: ```suggestion
We performed a broad-reaching public BIDS community survey before this project started and a narrower website-specific survey at the 60% mark in this project’s timeline to assess the improvements being made to online materials. Data collected on the overall utility of the old vs the new website indicated the new site was rated significantly more positively in 82% of responses overall. Key improvements noted were acceleration in accessing info, more intuitive navigation in both the search bar and the navigation tabs, and the feedback reporting page. 
```
@ericearl is 83% roughly right?
